politeiawww/convert.go

```diff
@@ -1541,19 +1541,32 @@ func convertStartVoteToCMS(sv cms.StartVote) cmsplugin.StartVote {
 
 }
 
-func filterDomainInvoice(inv *cms.InvoiceRecord) cms.InvoiceRecord {
+func filterDomainInvoice(inv *cms.InvoiceRecord, requestedDomain int) cms.InvoiceRecord {
 	inv.Files = nil
 	inv.Input.ContractorContact = ""
 	inv.Input.ContractorLocation = ""
 	inv.Input.ContractorName = ""
 	inv.Input.ContractorRate = 0
 	inv.Input.PaymentAddress = ""
 
-	for i, li := range inv.Input.LineItems {
-		li.Expenses = 0
-		li.SubRate = 0
-		inv.Input.LineItems[i] = li
+	filteredLineItems := make([]cms.LineItemsInput, 0, len(inv.Input.LineItems))
+	for _, li := range inv.Input.LineItems {
+		// Get the Supported CMS Domain from API
+		var cmsDomain cms.AvailableDomain
+		for _, domain := range cms.PolicySupportedCMSDomains {
+			if int(domain.Type) == requestedDomain {
+				cmsDomain = domain
```

Comment:
I guess PolicySupportedCMSDomains should never return two domains with the same type, but for extra safety this for loop might want a break.